### PR TITLE
Added transform parameter to the to_geotiff function

### PR DESCRIPTION
**Pull request by cath34**

### CODE REVIEW

File: notebooks/03. Home Range & Movescape/EcoGraph.ipynb [hunk 1]
```diff
@@ -69,7 +70,7 @@
     "    from google.colab import drive\n",
     "\n",
     "    drive.mount(\"/content/drive/\", force_remount=True)\n",
-    "    output_dir = os.path.join(\"/content/drive/MyDrive/\", output_dir)\n",
+    "    output_dir = os.path.join(output_dir, \"/content/drive/MyDrive/\")\n",
```
Comment (walljcg) [inline notebooks/03. Home Range & Movescape/EcoGraph.ipynb:73]: this is backwards and was correct before- the output_dir should be appended to the /content/drive/MyDrive base folder

**Review by walljcg — CHANGES_REQUESTED** [review]
This is great - thanks. I just had one comment about the output_dir folder path for Google Drive. And also, I don't think we should be committing test outputs to the repo: tests/test_output/salif_degree_mean_std.feather

File: notebooks/03. Home Range & Movescape/EcoGraph.ipynb (hunk 1 above)
Comment (kulits) [reply notebooks/03. Home Range & Movescape/EcoGraph.ipynb:73]: Please do not hard code Google Drive paths in the notebooks. They should be able to be run locally.

Comment (cath34) [reply notebooks/03. Home Range & Movescape/EcoGraph.ipynb]: > This is great - thanks. I just had one comment about the output_dir folder path for Google Drive. And also, I don't think we should be committing test outputs to the repo: tests/test_output/salif_degree_mean_std.feather

The test_output folder contains files that allow you to verify that the output of some Ecoscope functions are those you would expect. I added a test that checks if the function output is the appropriate transformed GeoTIFF, so I added the expected output (in feather format) to the test_output folder in order to check with geopandas.testing if the function output was okay

Comment (walljcg) [reply notebooks/03. Home Range & Movescape/EcoGraph.ipynb]: Ok, yes that makes sense

**Review by walljcg — APPROVED** [review]

**Pull request merged**